build/commands/scripts/commands.js

```diff
@@ -30,6 +30,18 @@ function commaSeparatedList(value, dummyPrevious) {
   return value.split(',')
 }
 
+// Use this wrapper function instead of JavaScript's parseInt() with option()
+// when defining integer optional parameters, or the default value might get
+// passed as well into the radix parameter of parseInt(), causing wrong results.
+// https://github.com/brave/brave-browser/issues/13724
+function parseInteger(string) {
+  // As per the spec [1], passing radix:0 to parseInt() will force parsing to
+  // understand the string passed as a decimal number unless it'sprefixed with
+  // '0' (octal) or '0x' (hexadecimal), so let's be explicit about that.
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
+  return parseInt(string, 0)
```

Comment:
Should we just use radix 10 here?

Reply:
I left it like this to allow for more flexibility since this way you could use octal or hexadecimal, but at the same time I doubt anyone will want to do that to specify a timeout, so I'm happy to make it explicitly decimal.

Reply:
Unless we currently have a job or something that uses hex (I doubt it, but I wouldn't really know for sure), we should probably just use 10.

Reply:
Agreed. Fixed now, thanks!